tests/performance.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/env bash
```

Comment:
I think this should be `#!/bin/env sh` as i don't use any bashism and therefore all posix complicant shell can run this script

Reply:
right